python/lsst/dax/apdb/apdbSql.py

```diff
@@ -160,15 +160,15 @@ class ApdbSqlConfig(ApdbConfig):
     htm_index_column = Field[str](
         default="pixelId", doc="Name of a HTM index column for DiaObject and DiaSource tables"
     )
-    ra_dec_columns = ListField[str](default=["ra", "decl"], doc="Names ra/dec columns in DiaObject table")
+    ra_dec_columns = ListField[str](default=["ra", "decl"], doc="Names of ra/dec columns in DiaObject table")
     dia_object_columns = ListField[str](
         doc="List of columns to read from DiaObject, by default read all columns", default=[]
     )
     prefix = Field[str](doc="Prefix to add to table names and index names", default="")
     namespace = Field[str](
         doc=(
-            "Namespace or schema name for all tables in APDB database. "
-            "Presently only makes sense for PostgresQL backend. "
+            "Schema name for all tables in APDB database. "
```

Comment:
I'd recommend keeping this as "namespace"; PostgreSQL's insistence on calling this "schema" is a persistent source of confusion in the AP team, since we also talk about schemas in the "set of tables" sense. This is especially relevant given the `*schema_file` and `schema_name` config fields present in this class.

Reply:
Hmm, I know that `schema` is confusing, but it is the only correct term, and it's not only Postgres, other databases have schemas as well. I borrowed `namespace` from middleware land, and it is fine as a concept there, but in this context we should stick to the database speak. Having said this, I'll restore `namespace`, if it helps you to un-confuse things (but I'm sure it will confuse other people).